### ISSUE TITLE
fix: possibly wrong variable name of  ipython_tb_clear_frames()

### ISSUE THREAD

**Issue by tianjianjiang** [stas00/ipyexperiments#7]
https://github.com/stas00/ipyexperiments/blob/0.1.26/ipyexperiments/utils/ipython.py#L47
```
    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        if not IS_IN_IPYTHON:
            return func(*args, **kwargs)

        try:
            return func(*args, **kwargs)
        except:
            type, val, tb = sys.exc_info()
            traceback.clear_frames(exc_tb)  # here
```

**Comment by stas00** [comment]
You're flagging that it should be:
```
traceback.clear_frames(tb)  #
```
correct?

**Comment by tianjianjiang** [comment]
> You're flagging that it should be:
> 
> ```
> traceback.clear_frames(tb)  #
> ```
> 
> correct?

Hi @stas00,

Yes, I guess. Unless there is something going on with a global variable (somewhere below that line). Thank you for your swift reply!

**Comment by stas00** [comment]
This should be fixed now and I made a new release 0.1.28 so if you update your pip or conda it should work correctly now.

Thank you again for flagging the error, @tianjianjiang!
